legal-api/src/legal_api/services/warnings/business/business_checks/business.py

```diff
@@ -28,11 +29,14 @@ def check_business(business: any) -> list:
              Business.LegalTypes.PARTNERSHIP.value):
         result = firms_check(business)
     elif business.legal_type in \
-        (Business.LegalTypes.BC_CCC,
+        (Business.LegalTypes.BC_CCC.value,
          Business.LegalTypes.BC_ULC_COMPANY.value,
          Business.LegalTypes.COMP.value,
          Business.LegalTypes.BCOMP.value
          ):
```

Comment:
can replace this corp list with [CORPS](https://github.com/bcgov/lear/blob/acdc05ccff4e56131790e1cc650b42df95babfc9/legal-api/src/legal_api/models/business.py#L124-L133) list in business model.